.github/workflows/templates/ci.yml.jinja

```diff
@@ -313,6 +313,8 @@
   <%- include "test-salt.yml.jinja" %>
   <%- endif %>
 
+  <%- if skip_test_coverage_check == "false" or "skip_code_coverage" in skip_test_coverage_check %>
```

Comment:
Are we sure that the string `skip_code_coverage` is not present in `skip_test_coverage_check` when we don't want to skip?

Reply:
This is what the default's are:

```
(py-3.10)  ch3ll@megan-precision5550  ~/git/salt   fix_coverage  grep -ri "skip_test_coverage_check" .github | grep set
.github/workflows/templates/staging.yml.jinja:<%- set skip_test_coverage_check = "true" %>
.github/workflows/templates/nightly.yml.jinja:<%- set skip_test_coverage_check = skip_test_coverage_check|default("false") %>
.github/workflows/templates/scheduled.yml.jinja:<%- set skip_test_coverage_check = "false" %>
.github/workflows/templates/layout.yml.jinja:<%- set skip_test_coverage_check = skip_test_coverage_check|default("${{ fromJSON(needs.prepare-workflow.outputs.testrun)['skip_code_coverage'] }}") %>
```

I interpret that to mean the only options currently set for `skip_test_coverage_check` are:
"true"
"false"
${{ fromJSON(needs.prepare-workflow.outputs.testrun)['skip_code_coverage'] }}"
I can improve that `in` check and add more of the fromJSON call.

Reply:
Also, as far as I understand, this jinja would only be called during pre-commit and the results are exactly what I want. I don't want code coverage to run on staging releases. Let me know if you want me to improve the `in` check or not. Or improve this in any other way.

Reply:
I believe those are all supposed to be booleans or True/false as strings

Reply:
But if the current check unlocks 3006.3, go for it, and we can narrow the logic after release.

Reply:
Yes, except for the one in layout.yml.jinja is this `${{ fromJSON(needs.prepare-workflow.outputs.testrun)['skip_code_coverage'] }}"` by default. Which we want to keep that so you can add the coverage lable on PRs. I don't see it setting boolean, only `true/false` strings. I might be missing something as to where we are defining those jinja variables though.